indico/modules/events/papers/templates/emails/added_to_reviewing_team.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'events/papers/emails/base.html' %}
+
+{% block subject_message %}Added in paper reviewing team{% endblock %}
+
+{% block header_subtitle %}Added in paper reviewing team{% endblock %}
+
+{% block content %}
+    <p>Dear {{ receiver.first_name }},</p>
+    <p>You have been chosen as a <strong>{{ role.title }}</strong> for the conference <strong>{{ event.title }}</strong> in order to help with the paper reviewing process.</p>
+    {% if role.name == 'judge' %}
+        <p>Once there are some papers for you to judge, you will be able to do so in the judging area page:</p>
+        <p>
+            <a href="{{ url_for('papers.papers_list', event, _external=true) }}">
+                {{- url_for('papers.papers_list', event, _external=true) -}}
```

Comment:
FYI this applies to all the endpoints defined in the `for prefix, is_management in ...` loop, but I believe `papers_list` is the only one you are using.